docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
             - OAUTHLIB_INSECURE_TRANSPORT=true
             - DJANGO_DEFAULT_SAMPLE_FHIR_ID="-20140000008325"
             - DJANGO_SECURE_SESSION=False
-            - FHIR_URL="https://prod-sbx.bfd.cms.gov/v1/fhir/"
+            - FHIR_URL="https://prod-sbx.bfd.cms.gov"
```

Comment:
@JFU-GIT I think this is OK, but we should check to see if the "FHIR_URL" ENV variable is being used via CBC Jenkins and/or in vault settings.

Reply:
yea, with v1 and v2 co-exists, might need to make FHIR_URL free of version component across settings.